lib/datadog/open_feature/binding/configuration.rb

```diff
@@ -0,0 +1,292 @@
+# frozen_string_literal: true
+
+module Datadog
+  module OpenFeature
+    module Binding
+      # Variation types supported by UFC
+      module VariationType
+        STRING = 'STRING'
+        INTEGER = 'INTEGER'
+        NUMERIC = 'NUMERIC'
+        BOOLEAN = 'BOOLEAN'
+        JSON = 'JSON'
+      end
+
+      # Condition operators for rule evaluation
+      module ConditionOperator
+        MATCHES = 'MATCHES'
+        NOT_MATCHES = 'NOT_MATCHES'
+        GTE = 'GTE'
+        GT = 'GT'
+        LTE = 'LTE'
+        LT = 'LT'
+        ONE_OF = 'ONE_OF'
+        NOT_ONE_OF = 'NOT_ONE_OF'
+        IS_NULL = 'IS_NULL'
+      end
+
+      # Assignment reasons returned in ResolutionDetails
+      module AssignmentReason
```

Comment:
I think it also would improve readability and code navigation, if those modules would be extracted as separate files